about.html

```diff
@@ -0,0 +1,138 @@
+                </ul>
+            </div>
+    
+            <div class="main-page-campaign">SB Campaign</div>
+    
+        </nav>
+
+        <div class="main-page-content">
+            <h3 class="main-page-subtitle">"Web3! Open to everyone"</h3>
+            <h1 class="main-page-title">SMARTBUILD GLOBAL <br> MEETING SUMMIT 2023</h1>
+            <hr class="about-hr">
+            <p class="main-page-paragraph">
+                SmartBuild is a blockchain company that connects many people around the world to share knowledge and most updated news
+                in the blockchain environment!
+                <br>
+                SmartBuild is aiming to summarize the evaluation of web1 web2 and web3 comprehensively. Participants
+                with various fields such as De-cent Blockchain, DeFi and De-cent Blockchain Oracle Network etc. Furthermore, will
+                discuss about the upcoming updates in web3 environment.
+                <br>
+                In the conference, several big-tech decentralized companies are going to present their newest products,
+                updates and features to the community!
+                <br>
+                In previous years, some blockchain events similar to SmartBuid conference took place in various locations such as
+                New York and Istanbul. 
+                <br>
+                The unique part of the SmartBuild conference is the tickets are totally free! Tickets are being shared with a Smart
+                Contract lottery and people have equal chances to join!
+             
+            </p>
+
+            <p class="main-page-contact">
+                Contact us if you have any question! 
+            </p>
+
+            <a class="mail-anchor" href="#"> <p class="mail"> smartbuild2023@dev.com</p></a>
+            <br><br><br>
+           
+        </div>
+    </section>
+
+    <hr class="hr">
+    
+    <section class="logo-section">
+        <p class="logo-creative">Creative Commons SmartBuild Summit 2023 Logo </p>   
+        <hr class="about-hr">
+        <p class="logo-competition">The logo of SmartBuild was decided with a competition from 13 April to 15 April</p>
+        <div class="logo-section-image"><img src="img/smartbuild.jpg" alt=""></div>
+        <hr class="separation-hr">
+        <p class="logo-events-look">Take a look at last 2 similar events</p>
+        <hr class="about-hr">
+        <p class="logo-competition">Look at the similar past 2 events took place</p>
+        <p class="logo-competition">on various locations.</p>
+
+
+        <div class="card-container">
+
+            <div class="card-item"><img class="first-event" src="img/smartcon.jpg" alt=""></div>
+            <div class="card-item"><img class="second-event" src="img/istanbul-summit.jpg" alt=""></div>
+         
+             <!--
+            <div class="card-item first-event"></div>
+            <div class="card-item second-event"></div>
+              -->
+
+        </div>
```

Comment:
- [ ] Please know that these images should have an overlay using the main color `#ec5242`, so that it matches the design provided, yours right now looks like this:

   ![image](https://user-images.githubusercontent.com/20567503/197231607-22aa4a5b-48d6-480a-9e05-7256475a9b0b.png)
   
   The original design is like this:
   
   ![image](https://user-images.githubusercontent.com/20567503/197231685-6f4389e8-4d9c-47a8-9abd-0963431d95b4.png)